docs/data-model.md

```diff
@@ -0,0 +1,29 @@
+# alerts_concierge Data Model
+
+## User-Driven
+
+In alerts_concierge, *trips* (referred to in the UI as “subscriptions”) represent an entire commute, potentially including transfers and a return journey.
+
+A trip is composed of multiple *subscriptions*, representing an individual leg of a commute; subscriptions have a single time range, a single route, and a single pair of stops where they start and end.
```

Comment:
note: If I remember correctly, bus subscriptions work slightly differently and do not include stop and end stops. (I think the reasoning is that we don't need to ask the rider for more than that since, in practice, bus alerts are only created for entire routes.)

There's also a thing where you can have "parallel" bus routes in your trip. I personally use this: when I'm traveling to the office, I can take either the 131 or the 137 to get to the Orange Line, so I put both into my subscription and get alerts for either route.

Reply:
Ah, yeah, good catch. I'm not sure if that reasoning is quite accurate, though - bus detours, for example, can specify a range of stops that may or may not include the segment a rider cares about.

Reply:
That's true, good point. Maybe it was more about making it less cumbersome for riders to get through the subscription process. It was a long time ago and my memory isn't always great… (But if either of you are ever interested, Jessie R was the designer on the project and might also have some valuable recollections.)